### ISSUE TITLE
[member-delimiter-style] Severity should be one of the following...

### ISSUE THREAD

**Issue by kg-currenxie** [typescript-eslint/typescript-eslint#663]
My config:

```js
module.exports = {
  env: {
    browser: true,
    es6: true,
    node: true,
    jest: true,
  },
  parser: '@typescript-eslint/parser', // Specifies the ESLint parser
  extends: [
    'plugin:react/recommended', // uses react-specific linting rules
    'plugin:@typescript-eslint/recommended', // uses typescript-specific linting rules
    'plugin:import/errors',
    'plugin:import/warnings',
    'plugin:import/typescript',
    'plugin:prettier/recommended', // enables eslint-plugin-prettier and eslint-config-prettier
    'prettier/react', // disables react-specific linting rules that conflict with prettier
    'prettier/@typescript-eslint',  // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
  ],
  plugins: [
    'simple-import-sort',
    'react-hooks',
  ],
  parserOptions: {
    ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
    sourceType: 'module', // Allows for the use of imports
    ecmaFeatures: {
      jsx:  true,  // Allows for the parsing of JSX
    },
  },
  settings:  {
    react:  {
      version: 'detect', // Tells eslint-plugin-react to automatically detect the version of React to use
    },
    'import/resolver': {
      // use <root>/tsconfig.json
      'typescript': {},
    },
  },
  rules: {
    // https://eslint.org/docs/4.0.0/rules/
    // Some base rule also needs to be changed to reflect rules below, as it can reporting incorrect errors
    'array-bracket-spacing': ['error', 'never'],
    'computed-property-spacing': ['error', 'never'],
    'func-call-spacing': 'off',
    'no-case-declarations': 'error',
    'no-multiple-empty-lines': ['error', { 'max': 1, 'maxEOF': 1, 'maxBOF': 0 }],
    'no-param-reassign': 'error',
    'no-return-assign': 'error',
    'no-shadow': 'error',
    'no-undef': 'error',
    'no-underscore-dangle' : ['off', 'always'],
    'object-curly-spacing': ['error', 'always'],
    'prefer-const': 'error',
    'quotes': ['error', 'single', 'avoid-escape'],
    'react/prefer-es6-class': ['off'],
    'space-in-parens': ['error', 'never'],
    'template-curly-spacing': ['error', 'never'],

    // Plugin
    // https://github.com/benmosher/eslint-plugin-import/tree/master/docs/rules
    'import/imports-first': ['error', 'absolute-first'],
    'import/newline-after-import': 'error',
    'import/named': 0,
    'import/no-named-as-default': 0,
    'import/no-unresolved' : 0,
    'import/newline-after-import': 2,

    // Plugin
    // https://github.com/lydell/eslint-plugin-simple-import-sort
    'simple-import-sort/sort': 'error',

    // https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin/docs/rules
    '@typescript-eslint/explicit-function-return-type': 'off',
    '@typescript-eslint/explicit-member-accessibility': 'off',
    '@typescript-eslint/func-call-spacing': 'error',
    // '@typescript-eslint/indent': ['error', 2],
    '@typescript-eslint/interface-name-prefix': 'warn',
    '@typescript-eslint/member-delimiter-style': {
      'multiline': {
          'delimiter': 'none',
          'requireLast': true
      },
      'singleline': {
          'delimiter': 'semi',
          'requireLast': false
      },
    },
    '@typescript-eslint/no-explicit-any': 'always',
    '@typescript-eslint/no-empty-interface': ['error', { 'allowSingleExtends': true }],
    '@typescript-eslint/no-extraneous-class': 'error',
    '@typescript-eslint/no-non-null-assertion': 'off',
    '@typescript-eslint/no-this-alias': 'error',
    '@typescript-eslint/no-use-before-define': 'off',
    '@typescript-eslint/no-unused-vars': ['error', {
      'vars': 'all',
      'args': 'all',
      'argsIgnorePattern': '^_',
      'ignoreRestSiblings': false,
    }],
    '@typescript-eslint/no-useless-constructor': 'error',
    '@typescript-eslint/semi': ['error', 'never'],
    '@typescript-eslint/type-annotation-spacing': 'error',

    // https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules
    'react/boolean-prop-naming': 'error',
    'react/display-name': 'off',
    'react/jsx-boolean-value': 'error',
    'react/jsx-closing-bracket-location': ['error', { 'selfClosing': 'line-aligned', nonEmpty: 'line-aligned' }],
    'react/jsx-closing-tag-location': 'error',
    'react/jsx-curly-brace-presence': ['error', { props: 'never', children: 'never' }],
    'react/jsx-curly-spacing': ['error', { 'when': 'never', 'allowMultiline': true }],
    'react/jsx-equals-spacing': ['error', 'never'],
    'react/jsx-filename-extension': ['error', { extensions: ['.ts', '.tsx'] }],
    'react/jsx-fragments': 'error',
    'react/jsx-handler-names': 'off',
    'react/jsx-key': 'error',
    'react/jsx-no-undef': 'error',
    'react/jsx-pascal-case': 'error',
    'react/jsx-props-no-multi-spaces': 'error',
    'react/jsx-tag-spacing': 'error',
    'react/jsx-uses-react': 'error',
    'react/jsx-wrap-multilines': ['error', 'parens-new-line'],
    'react/no-access-state-in-setstate': 'error',
    'react/no-array-index-key': 'error',
    'react/no-deprecated': 'error',
    'react/no-direct-mutation-state': 'error',
    'react/no-find-dom-node': 'error',
    'react/no-string-refs': ['error', { 'noTemplateLiterals': true }],
    'react/prop-types': 'off',
    'react/require-render-return': 'error',
    'react/self-closing-comp': 'error',
    'react/sort-comp': 'error',

    // https://www.npmjs.com/package/eslint-plugin-react-hooks
    'react-hooks/rules-of-hooks': 'error',
    'react-hooks/exhaustive-deps': 'warn',
  }
}
```

**Expected Result**
Should be able to use complex rule for https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-delimiter-style.md

**Actual Result**
```
Configuration for rule "@typescript-eslint/member-delimiter-style" is invalid:
	Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '{  multiline: { delimiter: "none", requireLast: true },  singleline: { delimiter: "semi", requireLast: false }}').
```

Debug output

```
  eslint:cli CLI args: [ '--ext', '.ts,.js', 'src', '--debug' ] +0ms
  eslint:cli Running on files +5ms
  eslint:ignored-paths baseDir = "/Users/kristiangerardsson/www/episodecalendar.com/native_app" +0ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "/node_modules/*"
  eslint:ignored-paths   cooked   = "/node_modules/*" +1ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "/bower_components/*"
  eslint:ignored-paths   cooked   = "/bower_components/*" +0ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = ".*"
  eslint:ignored-paths   cooked   = ".*" +0ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "!../"
  eslint:ignored-paths   cooked   = "!../" +1ms
  eslint:ignored-paths Looking for ignore file in /Users/kristiangerardsson/www/episodecalendar.com/native_app +0ms
  eslint:ignored-paths Could not find ignore file in cwd +0ms
  eslint:ignored-paths baseDir = "/Users/kristiangerardsson/www/episodecalendar.com/native_app" +1ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "/node_modules/*"
  eslint:ignored-paths   cooked   = "/node_modules/*" +0ms
  eslint:ignored-paths addPatternRelativeToCwd:
  eslint:ignored-paths   original = "/bower_components/*"
  eslint:ignored-paths   cooked   = "/bower_components/*" +0ms
  eslint:ignored-paths Looking for ignore file in /Users/kristiangerardsson/www/episodecalendar.com/native_app +0ms
  eslint:ignored-paths Could not find ignore file in cwd +0ms
  eslint:file-enumerator Start to iterate files: [ 'src' ] +0ms
  eslint:file-enumerator Directory: /Users/kristiangerardsson/www/episodecalendar.com/native_app/src +1ms
  eslint:ignored-paths contains: +4ms
  eslint:ignored-paths   target   = "/Users/kristiangerardsson/www/episodecalendar.com/native_app/src/" +0ms
  eslint:ignored-paths   base     = "/Users/kristiangerardsson/www/episodecalendar.com/native_app" +0ms
  eslint:ignored-paths   relative = "src/" +0ms
  eslint:ignored-paths   result   = false +0ms
  eslint:file-enumerator Enter the directory: /Users/kristiangerardsson/www/episodecalendar.com/native_app/src +2ms
  eslint:cascading-config-array-factory Load config files for /Users/kristiangerardsson/www/episodecalendar.com/native_app/src. +0ms
  eslint:cascading-config-array-factory No cache found: /Users/kristiangerardsson/www/episodecalendar.com/native_app/src. +0ms
  eslint:config-array-factory Config file not found on /Users/kristiangerardsson/www/episodecalendar.com/native_app/src +0ms
  eslint:cascading-config-array-factory No cache found: /Users/kristiangerardsson/www/episodecalendar.com/native_app. +1ms
  eslint:config-array-factory Loading JS config file: /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Config file found: /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +2ms
  eslint:config-array-factory Loading {extends:"plugin:react/recommended"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +14ms
  eslint:config-array-factory Loading plugin "react" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loaded: eslint-plugin-react@7.14.2 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-react/index.js) +2ms
  eslint:config-array-factory Loading plugin "react" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-react/index.js +163ms
  eslint:config-array-factory Loaded: eslint-plugin-react@7.14.2 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-react/index.js) +0ms
  eslint:config-array-factory Loading {extends:"plugin:@typescript-eslint/recommended"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loading plugin "@typescript-eslint" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loaded: @typescript-eslint/eslint-plugin@1.11.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/eslint-plugin/dist/index.js) +2ms
  eslint:config-array-factory Loading {extends:"./configs/base.json"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/eslint-plugin/dist/index.js +438ms
  eslint:config-array-factory package.json was not found: Cannot find module './configs/base.json/package.json'
Require stack:
- /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/eslint-plugin/dist/index.js +0ms
  eslint:config-array-factory Loaded: ./configs/base.json (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/eslint-plugin/dist/configs/base.json) +0ms
  eslint:config-array-factory Loading JSON config file: /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/eslint-plugin/dist/configs/base.json +0ms
  eslint:config-array-factory Loading parser "@typescript-eslint/parser" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/eslint-plugin/dist/configs/base.json +1ms
  eslint:config-array-factory Loaded: @typescript-eslint/parser@1.11.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/parser/dist/parser.js) +1ms
  eslint:config-array-factory Loading plugin "@typescript-eslint" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/eslint-plugin/dist/configs/base.json +34ms
  eslint:config-array-factory Loaded: @typescript-eslint/eslint-plugin@1.11.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/eslint-plugin/dist/index.js) +0ms
  eslint:config-array-factory Loading {extends:"plugin:import/errors"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loading plugin "import" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loaded: eslint-plugin-import@2.18.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-import/lib/index.js) +1ms
  eslint:config-array-factory Loading plugin "import" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-import/lib/index.js +112ms
  eslint:config-array-factory Loaded: eslint-plugin-import@2.18.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-import/lib/index.js) +0ms
  eslint:config-array-factory Loading {extends:"plugin:import/warnings"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loading plugin "import" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loaded: eslint-plugin-import@2.18.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-import/lib/index.js) +0ms
  eslint:config-array-factory Loading plugin "import" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-import/lib/index.js +1ms
  eslint:config-array-factory Loaded: eslint-plugin-import@2.18.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-import/lib/index.js) +0ms
  eslint:config-array-factory Loading {extends:"plugin:import/typescript"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loading plugin "import" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loaded: eslint-plugin-import@2.18.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-import/lib/index.js) +0ms
  eslint:config-array-factory Loading {extends:"plugin:prettier/recommended"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loading plugin "prettier" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loaded: eslint-plugin-prettier@3.1.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js) +1ms
  eslint:config-array-factory Loading {extends:"prettier"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js +3ms
  eslint:config-array-factory Loaded: eslint-config-prettier@6.0.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-config-prettier/index.js) +1ms
  eslint:config-array-factory Loading JS config file: /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-config-prettier/index.js +0ms
  eslint:config-array-factory Loading plugin "prettier" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js +1ms
  eslint:config-array-factory Loaded: eslint-plugin-prettier@3.1.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js) +0ms
  eslint:config-array-factory Loading {extends:"prettier/react"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory package.json was not found: Cannot find module 'eslint-config-prettier/react/package.json'
Require stack:
- /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +1ms
  eslint:config-array-factory Loaded: eslint-config-prettier/react (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-config-prettier/react.js) +0ms
  eslint:config-array-factory Loading JS config file: /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-config-prettier/react.js +0ms
  eslint:config-array-factory Loading {extends:"prettier/@typescript-eslint"} relative to /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory package.json was not found: Cannot find module 'eslint-config-prettier/@typescript-eslint/package.json'
Require stack:
- /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +20ms
  eslint:config-array-factory Loaded: eslint-config-prettier/@typescript-eslint (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-config-prettier/@typescript-eslint.js) +0ms
  eslint:config-array-factory Loading JS config file: /Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-config-prettier/@typescript-eslint.js +0ms
  eslint:config-array-factory Loading parser "@typescript-eslint/parser" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +1ms
  eslint:config-array-factory Loaded: @typescript-eslint/parser@1.11.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/@typescript-eslint/parser/dist/parser.js) +0ms
  eslint:config-array-factory Loading plugin "simple-import-sort" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +0ms
  eslint:config-array-factory Loaded: eslint-plugin-simple-import-sort@4.0.0 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-simple-import-sort/src/index.js) +1ms
  eslint:config-array-factory Loading plugin "react-hooks" from /Users/kristiangerardsson/www/episodecalendar.com/native_app/.eslintrc.js +3ms
  eslint:config-array-factory Loaded: eslint-plugin-react-hooks@1.6.1 (/Users/kristiangerardsson/www/episodecalendar.com/native_app/node_modules/eslint-plugin-react-hooks/index.js) +1ms
  eslint:cascading-config-array-factory No cache found: /Users/kristiangerardsson/www/episodecalendar.com. +806ms
  eslint:config-array-factory Config file not found on /Users/kristiangerardsson/www/episodecalendar.com +2ms
  eslint:cascading-config-array-factory No cache found: /Users/kristiangerardsson/www. +0ms
  eslint:config-array-factory Config file not found on /Users/kristiangerardsson/www +1ms
  eslint:cascading-config-array-factory No cache found: /Users/kristiangerardsson. +1ms
  eslint:cascading-config-array-factory Stop traversing because of considered root. +0ms
  eslint:rules Loading rule 'camelcase' (remaining=265) +0ms
  eslint:rules Loading rule 'indent' (remaining=264) +5ms
  eslint:rules Loading rule 'no-array-constructor' (remaining=263) +20ms
  eslint:rules Loading rule 'no-unused-vars' (remaining=262) +5ms
  eslint:rules Loading rule 'no-use-before-define' (remaining=261) +3ms
  eslint:rules Loading rule 'arrow-body-style' (remaining=260) +6ms
  eslint:rules Loading rule 'curly' (remaining=259) +1ms
  eslint:rules Loading rule 'lines-around-comment' (remaining=258) +1ms
  eslint:rules Loading rule 'max-len' (remaining=257) +1ms
  eslint:rules Loading rule 'no-confusing-arrow' (remaining=256) +1ms
  eslint:rules Loading rule 'no-mixed-operators' (remaining=255) +0ms
  eslint:rules Loading rule 'no-tabs' (remaining=254) +1ms
  eslint:rules Loading rule 'no-unexpected-multiline' (remaining=253) +1ms
  eslint:rules Loading rule 'prefer-arrow-callback' (remaining=252) +1ms
  eslint:rules Loading rule 'quotes' (remaining=251) +1ms
  eslint:rules Loading rule 'array-bracket-newline' (remaining=250) +1ms
  eslint:rules Loading rule 'array-bracket-spacing' (remaining=249) +2ms
  eslint:rules Loading rule 'array-element-newline' (remaining=248) +0ms
  eslint:rules Loading rule 'arrow-parens' (remaining=247) +1ms
  eslint:rules Loading rule 'arrow-spacing' (remaining=246) +1ms
  eslint:rules Loading rule 'block-spacing' (remaining=245) +1ms
  eslint:rules Loading rule 'brace-style' (remaining=244) +0ms
  eslint:rules Loading rule 'comma-dangle' (remaining=243) +1ms
  eslint:rules Loading rule 'comma-spacing' (remaining=242) +1ms
  eslint:rules Loading rule 'comma-style' (remaining=241) +0ms
  eslint:rules Loading rule 'computed-property-spacing' (remaining=240) +1ms
  eslint:rules Loading rule 'dot-location' (remaining=239) +1ms
  eslint:rules Loading rule 'eol-last' (remaining=238) +1ms
  eslint:rules Loading rule 'func-call-spacing' (remaining=237) +0ms
  eslint:rules Loading rule 'function-paren-newline' (remaining=236) +1ms
  eslint:rules Loading rule 'generator-star-spacing' (remaining=235) +1ms
  eslint:rules Loading rule 'implicit-arrow-linebreak' (remaining=234) +0ms
  eslint:rules Loading rule 'indent-legacy' (remaining=233) +1ms
  eslint:rules Loading rule 'jsx-quotes' (remaining=232) +1ms
  eslint:rules Loading rule 'key-spacing' (remaining=231) +1ms
  eslint:rules Loading rule 'keyword-spacing' (remaining=230) +2ms
  eslint:rules Loading rule 'linebreak-style' (remaining=229) +2ms
  eslint:rules Loading rule 'multiline-ternary' (remaining=228) +1ms
  eslint:rules Loading rule 'newline-per-chained-call' (remaining=227) +1ms
  eslint:rules Loading rule 'new-parens' (remaining=226) +0ms
  eslint:rules Loading rule 'no-extra-parens' (remaining=225) +1ms
  eslint:rules Loading rule 'no-extra-semi' (remaining=224) +1ms
  eslint:rules Loading rule 'no-floating-decimal' (remaining=223) +4ms
  eslint:rules Loading rule 'no-mixed-spaces-and-tabs' (remaining=222) +1ms
  eslint:rules Loading rule 'no-multi-spaces' (remaining=221) +1ms
  eslint:rules Loading rule 'no-multiple-empty-lines' (remaining=220) +1ms
  eslint:rules Loading rule 'no-spaced-func' (remaining=219) +1ms
  eslint:rules Loading rule 'no-trailing-spaces' (remaining=218) +1ms
  eslint:rules Loading rule 'no-whitespace-before-property' (remaining=217) +1ms
  eslint:rules Loading rule 'nonblock-statement-body-position' (remaining=216) +1ms
  eslint:rules Loading rule 'object-curly-newline' (remaining=215) +1ms
  eslint:rules Loading rule 'object-curly-spacing' (remaining=214) +1ms
  eslint:rules Loading rule 'object-property-newline' (remaining=213) +1ms
  eslint:rules Loading rule 'one-var-declaration-per-line' (remaining=212) +2ms
  eslint:rules Loading rule 'operator-linebreak' (remaining=211) +1ms
  eslint:rules Loading rule 'padded-blocks' (remaining=210) +0ms
  eslint:rules Loading rule 'quote-props' (remaining=209) +1ms
  eslint:rules Loading rule 'rest-spread-spacing' (remaining=208) +1ms
  eslint:rules Loading rule 'semi' (remaining=207) +1ms
  eslint:rules Loading rule 'semi-spacing' (remaining=206) +1ms
  eslint:rules Loading rule 'semi-style' (remaining=205) +0ms
  eslint:rules Loading rule 'space-before-blocks' (remaining=204) +1ms
  eslint:rules Loading rule 'space-before-function-paren' (remaining=203) +1ms
  eslint:rules Loading rule 'space-in-parens' (remaining=202) +1ms
  eslint:rules Loading rule 'space-infix-ops' (remaining=201) +0ms
  eslint:rules Loading rule 'space-unary-ops' (remaining=200) +1ms
  eslint:rules Loading rule 'switch-colon-spacing' (remaining=199) +1ms
  eslint:rules Loading rule 'template-curly-spacing' (remaining=198) +1ms
  eslint:rules Loading rule 'template-tag-spacing' (remaining=197) +0ms
  eslint:rules Loading rule 'unicode-bom' (remaining=196) +1ms
  eslint:rules Loading rule 'wrap-iife' (remaining=195) +1ms
  eslint:rules Loading rule 'wrap-regex' (remaining=194) +14ms
  eslint:rules Loading rule 'yield-star-spacing' (remaining=193) +2ms
  eslint:rules Loading rule 'no-case-declarations' (remaining=192) +4ms
  eslint:rules Loading rule 'no-param-reassign' (remaining=191) +2ms
  eslint:rules Loading rule 'no-return-assign' (remaining=190) +2ms
  eslint:rules Loading rule 'no-shadow' (remaining=189) +1ms
  eslint:rules Loading rule 'no-undef' (remaining=188) +2ms
  eslint:rules Loading rule 'no-underscore-dangle' (remaining=187) +2ms
  eslint:rules Loading rule 'prefer-const' (remaining=186) +2ms
  eslint:rules Loading rule 'no-restricted-imports' (remaining=185) +5ms
Error: .eslintrc.js:
	Configuration for rule "@typescript-eslint/member-delimiter-style" is invalid:
	Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '{  multiline: { delimiter: "none", requireLast: true },  singleline: { delimiter: "semi", requireLast: false }}').

    at validateRuleOptions (/usr/local/Cellar/node/8.5.0/lib/node_modules/eslint/lib/shared/config-validator.js:138:19)
    at /usr/local/Cellar/node/8.5.0/lib/node_modules/eslint/lib/shared/config-validator.js:193:9
    at Array.forEach (<anonymous>)
    at validateRules (/usr/local/Cellar/node/8.5.0/lib/node_modules/eslint/lib/shared/config-validator.js:190:30)
    at validateConfigArray (/usr/local/Cellar/node/8.5.0/lib/node_modules/eslint/lib/shared/config-validator.js:337:9)
    at CascadingConfigArrayFactory._finalizeConfigArray (/usr/local/Cellar/node/8.5.0/lib/node_modules/eslint/lib/cli-engine/cascading-config-array-factory.js:392:13)
    at CascadingConfigArrayFactory.getConfigArrayForFile (/usr/local/Cellar/node/8.5.0/lib/node_modules/eslint/lib/cli-engine/cascading-config-array-factory.js:248:21)
    at FileEnumerator._iterateFilesRecursive (/usr/local/Cellar/node/8.5.0/lib/node_modules/eslint/lib/cli-engine/file-enumerator.js:408:49)
    at _iterateFilesRecursive.next (<anonymous>)
    at FileEnumerator.iterateFiles (/usr/local/Cellar/node/8.5.0/lib/node_modules/eslint/lib/cli-engine/file-enumerator.js:254:49)
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `1.11.0` |
| `@typescript-eslint/parser`        | `1.11.0` |
| `TypeScript`                       | `3.5.2` |
| `ESLint`                           | `6.0.1` |
| `node`                             | `12.4.0` |
| `yarn`                              | `1.3.2` |

**Comment by bradzacher** [comment]
please read the eslint documentation

https://eslint.org/docs/user-guide/configuring#configuring-rules
